src/App.js

```diff
@@ -53,20 +89,29 @@ function App() {
       </Navbar>
       <Container>
         <Row>
-          <h1>this is a thing now</h1>
+          {/* render grocery item */}
+          {mockFetchItems.map(item => (
```

Comment:
Actually, I noticed that each item is being rendered directly inside a row. 
I think we actually want an additional `<Col>` element to wrap each `GroceryItem`
like 
```
<Col>
<GroceryItem 
  itemTitle={item.title} 
  itemDesc={item.desc}
  imgUrl={item.img} 
  key={item.title}
/>
</Col>
```
Not a big diff atm but I think it's how bootstrap grids are meant to be used for resizing stuff. and we get nicer spacing with it for now. 

## With Col
![image](https://user-images.githubusercontent.com/7408875/66703069-3c899580-ed5a-11e9-8063-c1aafee682f5.png)

## Without Col
![image](https://user-images.githubusercontent.com/7408875/66703072-4f03cf00-ed5a-11e9-9fd7-6867b933c9a4.png)

Reply:
I used flexbox for component and grid for the app itself as recommended [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). It does make sense and certain makes UI layout more structured🍻

In the background section...

> _Flexbox layout is most appropriate to the components of an application, and small-scale layouts, while the Grid layout is intended for larger scale layouts._

Reply:
![I like it](https://media1.giphy.com/media/3o7aD8yI3nrBhk6z3a/giphy.gif)